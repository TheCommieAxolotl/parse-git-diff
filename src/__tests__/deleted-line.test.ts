import { getFixture } from './test-utils';
import parseGitDiff from '../parse-git-diff';

describe('deleted-line', () => {
  const fixture = getFixture('deleted-line');

  it('parse `deleted-line`', () => {
    expect(parseGitDiff(fixture)).toMatchSnapshot();
  });
});
