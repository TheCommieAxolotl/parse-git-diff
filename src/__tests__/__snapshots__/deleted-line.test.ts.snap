// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deleted-line parse \`deleted-line\` 1`] = `
Object {
  "files": Array [
    Object {
      "chunks": Array [
        Object {
          "changes": Array [
            Object {
              "content": "newfile",
              "lineAfter": 1,
              "lineBefore": 1,
              "type": "UnchangedLine",
            },
            Object {
              "content": "newline",
              "lineBefore": 2,
              "type": "DeletedLine",
            },
          ],
          "rangeAfter": Object {
            "lines": 1,
            "start": 1,
          },
          "rangeBefore": Object {
            "lines": 2,
            "start": 1,
          },
          "type": "Chunk",
        },
      ],
      "path": "rename.md",
      "type": "ChangedFile",
    },
  ],
  "type": "GitDiff",
}
`;
